lib/Doctrine/ORM/Exception/RepositoryException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Exception;
+
+/**
+ * This interface should be implemented by all exceptions in the Repository
```

Comment:
btw, this could be introduced as an interface directly, with the actual extensions extending from the old class (for BC) and implementing the new extension.